src/firefly/js/visualize/ui/FileUploadViewPanel.jsx

```diff
@@ -622,18 +622,22 @@ function SingleDataSet({type, desc, detailsModel, report, UNKNOWN_FORMAT, curren
     const showDetails= supported && detailsModel;
     const isUWSJobFile = isUWS(report);
     const jobUrl = report.parts[0].url;
-
     return (
-        <Stack {...{direction:'row', flex:'1 1 auto', justifyContent: showDetails || isUWSJobFile?'start':'center'}}>
-            <Box sx={{pt:4, pr:2, ml: isUWSJobFile?4:0}}>
+        <Stack {...{direction:'row', flex:'1 1 auto', justifyContent: showDetails || isUWSJobFile?'start':'center', position:'relative'}}>
+            <Box sx={{pt:4, pr:2, ml: isUWSJobFile?4:0, flexShrink:0}}>
```

Comment:
Same here.  Should use Stack's spacing to layout children instead of them adjusting padding and margin.